src/app/static/src/app/components/plots/table/Table.vue

```diff
@@ -7,12 +7,25 @@
                     <tr>
                         <th v-translate="'area'"></th>
                         <th v-for="f in filtersToDisplay" v-translate="f.label"></th>
-                        <th v-for="i in indicators">{{ i.name }}</th>
+                        <template v-for="i in indicators">
+                          <th>{{ i.name }}</th>
+                          <!-- ideally I would put "i.error_high_column"/""i.error_high_column"" as the column header 
+                          instead of "Upper"/"Lower" and use them to control the v-if statements here and for the rows below,
+                          but I'll need Rob to pass them into indicators for the choropleth/bubbleplot from the backend first.
+                          Currently they are only available for the indicators of the barchart. I'll also need to add a v-translate  
+                          for upper/lower-->
+                          <th v-if="filteredData[0].indicatorValues[i.indicator].upper">Upper</th>
```

Comment:
When I spoke with UNAIDS, I specifically asked about this. They said their preferences was having the uncertainty ranges in separate columns because it makes it easier to sort/filter that way.